lib/components/BraidProvider/BraidProvider.tsx

```diff
@@ -20,31 +26,47 @@ export const useBraidTheme = () => {
   return braidTheme;
 };
 
+export interface LinkComponentProps
+  extends AnchorHTMLAttributes<HTMLAnchorElement> {
+  href: string;
+}
+export type LinkComponent = ComponentType<LinkComponentProps>;
+const DefaultLinkComponent = (props: LinkComponentProps) => <a {...props} />;
```

Comment:
Should the default be a reset anchor tag, eg. `<Box component="a" />`, thoughts? I cant decide.